vm/ubpf_vm.c

```diff
@@ -128,6 +125,21 @@ ubpf_load(struct ubpf_vm *vm, const void *code, uint32_t code_len, char **errmsg
     return 0;
 }
 
+void
+ubpf_unload(struct ubpf_vm *vm)
```

Comment:
Since `ext_funcs` and `ext_func_names` aren't freed, maybe name this `ubpf_unload_code`?

Reply:
Sounds good, I'll push a new version soon fixing this and adding vm reuse tests